modules/serverless-subscription-event-delivery-config.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-subscription-event-delivery-config_{context}"]
+= Configuring event delivery failure parameters using subscriptions
+
+Developers can configure event delivery parameters for individual subscriptions by modifying the `delivery` settings for a `Subscription` object.
+
+.Example subscription YAML
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Subscription
+metadata:
+  name: <subscription_name>
+  namespace: <subscription_namespace>
+spec:
+  delivery:
+    deadLetterSink: <1>
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: <sink_name>
+    backoffDelay: <duration> <2>
+    backoffPolicy: <policy_type> <3>
+    retry: <integer> <4>
+----
+<1> Configuration settings to enable using a dead letter sink. This tells the subscription what happens to events that cannot be delivered to the subscriber.
++
+When this is configured, events that failed to be delivered are sent to the dead letter sink destination. The destination can be a Knative service or a URI.
+<2> You can set the `backoffDelay` delivery parameter to specify the time delay before an event delivery retry is attempted after a failure. The duration of the `backoffDelay` parameter is specified using the https://en.wikipedia.org/wiki/ISO_8601#Durations[ISO 8601] format.
```

Comment:
We could perhaps use realistic example of ISO8601 duration format here.. e.g.  "PT1S" for 1 second delay.